tests/Microsoft.Bot.Builder.Prompts.Tests/ConfirmPromptTests.cs

```diff
@@ -0,0 +1,217 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Same comment. Great tests. Let's make sure the functionality they are testing is in a clean PR.